k8s.io/README.md

```diff
@@ -25,6 +26,7 @@ Vanity URL(s)
 | PR Dashboard | https://pr-test.k8s.io | https://pr-test.kubernetes.io |
 | Pull requests | https://pr.k8s.io <br> https://prs.k8s.io | https://pr.kubernetes.io <br> https://prs.kubernetes.io |
 | Downloads | https://releases.k8s.io <br> https://rel.k8s.io | https://releases.kubernetes.io <br> https://rel.kubernetes.io |
+| Kubernetes SIGs | https://sigs.k8s.io | |
```

Comment:
I think we might also be lacking docs about the go get handling for k8s.io / sigs.k8s.io

Reply:
+1, I was working on a repo in kubernetes-sigs recently and found out that the import path is `sigs.k8s.io` by looking at the list of imports in a `.go` file. :/